### PR TITLE
CommunityBridge is LFX Mentorship now

### LINKED ISSUE #3241: LFX Mentorship is a new name for CommunityBridge
### What is the current behavior?

https://github.com/github/docs/blob/main/content/github/administering-a-repository/displaying-a-sponsor-button-in-your-repository.md#about-funding-files

### What changes are you suggesting?

Community Bridge is now called LFX (specifically, LFX Mentorship for the mentoring purposes) - https://lfx.linuxfoundation.org/blog/introducing-lfx-tools-for-everyone/

Please update the wording where necessary.

### PR DESCRIPTION
**Pull request by idvoretskyi**
Fixes https://github.com/github/docs/issues/3241

Signed-off-by: Ihor Dvoretskyi <ihor@linux.com>

### CODE REVIEW

File: content/github/administering-a-repository/displaying-a-sponsor-button-in-your-repository.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ You can add one username, package name, or project name per external funding pla
 
 Platform | Syntax
 -------- | -----
-[CommunityBridge](https://communitybridge.org) | `community_bridge: PROJECT-NAME`
+[LFX Mentorship (formerly CommunityBridge)](https://lfx.linuxfoundation.org/tools/mentorship) | `lfx_mentorship: PROJECT-NAME`
```
Comment (ethanpalm) [inline content/github/administering-a-repository/displaying-a-sponsor-button-in-your-repository.md:19]:
```suggestion
[LFX Mentorship (formerly CommunityBridge)](https://lfx.linuxfoundation.org/tools/mentorship) | `community_bridge: PROJECT-NAME`
```

The syntax for this badge is remaining the same, so we won't need to change it in this PR.

**Review by ethanpalm — CHANGES_REQUESTED** [review]
Thanks for opening this PR! The syntax for this sponsor badge is remaining the same, so I made one small comment. With that change, we should be ready to merge 🚀  😄

**Comment by idvoretskyi** [comment]
@ethanpalm addressed, thanks!

**Review by ethanpalm — APPROVED** [review]

**Comment by ethanpalm** [comment]
@idvoretskyi thanks for making that change! This looks great and I'll get it merged as soon as all checks are passing ✨

**Pull request merged**